packages/flight-icons/scripts/build-parts/generateBundleSVGJS.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2021, 2025
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import fs from 'fs-extra';
+import cheerio from 'cheerio';
+import prettier from 'prettier';
+import path from 'path';
+
+import { ConfigData } from '../@types/ConfigData';
+import { AssetsCatalog } from '../@types/AssetsCatalog';
+
+const prettierConfig = {
+    parser: 'typescript',
+    tabWidth: 4,
+    singleQuote: true,
+    trailingComma: 'none'
+} as const;
+
+const getSymbolFunction = (sourceSvg: string): string => {
+    const $ = cheerio.load(sourceSvg, { xmlMode: true });
+    const $svg = $('svg');
+    const viewBox = $svg.attr('viewBox') || '0 0 16 16';
+
+    $('[fill]').each((_i: number, el: unknown) => {
+        const val = $(el).attr('fill');
+        if (val && val !== 'none') {
+            $(el).attr('fill', 'currentColor');
+        }
+    });
+
+    const innerContent = $svg.html() ?? '';
+
+    return `function (id) {
+        return \`<symbol id="\${id}" viewBox="${viewBox}" fill="none" xmlns="http://www.w3.org/2000/svg"><g fill="currentColor">${innerContent}</g></symbol>\`;
+    }`;
+};
+
+export async function generateBundleSVGJS({ config, catalog }: { config: ConfigData, catalog: AssetsCatalog }): Promise<void> {
+    const tempSVGFolderPath = config.tempFolder;
+    const mappingFilePath = path.resolve(__dirname, '../../../../hds-carbon-icon-map.json');
+    const carbonIconsPath = path.resolve(__dirname, '../../node_modules/@carbon/icons/svg');
+
+    let mapping: Record<string, string> = {};
+    try {
+        mapping = await fs.readJSON(mappingFilePath);
+    } catch {
+        console.warn('⚠️ Map file not found.');
+    }
+
+    const outputFolder = `${config.mainFolder}/js-symbols`;
```

Comment:
for consistency with the other folder names
```suggestion
    const outputFolder = `${config.mainFolder}/symbols-js`;
```